package.json

```diff
@@ -6,9 +6,13 @@
   "engines": {
     "node": ">=16"
```

Comment:
I think this will go out before we're fully committed to node 18, will likely update it to 18 when typescript 5.1 goes out (and therefore will be a major version update)